core/tests/protractor/accessibility.js

```diff
@@ -49,6 +49,7 @@ describe('Cache Slugs', function() {
   it('should check that errors get logged for missing resources', function() {
     browser.get('/console_errors');
     var expectedErrors = [
+      'ckeditor-duplicated-modules',
```

Comment:
Should you drop this if you're handling it more generally in general.js?

Reply:
Hi @seanlip, this matches the count of expected errors with the count of error summed from getting the library page. I found this test to be independent of general.js correct @apb7 ?

Reply:
Ah, I think I misunderstood this. I think what you have is OK. Thanks!